src/gui/SetupTotpDialog.cpp

```diff
@@ -72,13 +92,25 @@ void SetupTotpDialog::setSeed(QString value)
     m_ui->seedEdit->setText(value);
 }
 
+void SetupTotpDialog::setSettings(quint8 digits) {
+    quint8 step = m_ui->stepSpinBox->value();
+
+    bool isDefault = step == QTotp::defaultStep &&
+        digits == QTotp::defaultDigits;
+    bool isSteam = digits == QTotp::ENCODER_STEAM;
```

Comment:
Maybe use brackets here to visually separate the parts of these statements better.